doc/tutorials/contributing-clion.md

```diff
@@ -0,0 +1,183 @@
+# Introduction
+
+In this tutorial, we will go through the steps necessary to contribute to
+Elektra to make it easier for you to get started. We will use a Unix based OS
+like Linux and CLion for development, but depending on where you want to
+contribute code, other IDEs will also be sufficient. Before you start, please
+read [this](/.github/CONTRIBUTING.md) to get familiar with the process of
+contributing to Elektra.
+
+## Prerequisites
+
+- Git ([link](https://git-scm.com/download)) or similar software
+- CLion ([link](https://www.jetbrains.com/clion/)) or similar software
+- GitHub-account ([link](https://github.com/))
+
+## Forking the repository
```

Comment:
Please use [title case](https://github.com/ElektraInitiative/libelektra/blob/master/doc/CODING.md#markdown-guidelines) for headings.

```suggestion
## Forking the Repository
```